packages/components/src/templates/next/layouts/Collection/CollectionClient.tsx

```diff
@@ -51,6 +53,12 @@ const CollectionClient = ({
     setCurrPage,
   } = useCollection({ items })
 
+  const articleContainerRef = useRef<HTMLDivElement>(null)
+  const onPageChange = () => {
+    articleContainerRef.current?.scrollIntoView({
+      block: "start",
+    })
```

Comment:
nit: maybe can do smooth scroll?

Reply:
didn't do it because the page change actually happens instantly. You see two different changes (scroll + content change) i thought poorer UX haha